src/notifications/entities/notification.entity.ts

```
@@ -0,0 +1,57 @@
+  body: string | null;
+
+  // Optional deep link, e.g. "/tasks/42" or "/contacts/17"
+  @Column({ nullable: true })
+  link: string | null;
+
+  @Column({ default: false })
+  isRead: boolean;
+
+  @CreateDateColumn()
+  createdAt: Date;
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)